app/views/people/_member.html.haml

```diff
@@ -3,6 +3,11 @@
     - if member.respond_to?(:full_name) && member.respond_to?(:graph_id)
       %li
         %a{:href => "/people/#{member.graph_id}"}= member.full_name
+
+        %li= "#{t('current_lord')}" if member.statuses.include?('Current Lord')
```

Comment:
Should these be some kind of helper methods? OR something like 'status' which returns a string? or array for occasions where they are both? If this is even a possible edge case?